client/scripts/views/pages/manage_staking/groups/group_list.ts

```diff
@@ -4,27 +4,37 @@ import { makeDynamicComponent } from 'models/mithril';
 import { ChainBase } from 'models';
 import Substrate from 'controllers/chain/substrate/main';
 import { IValidators } from 'controllers/chain/substrate/account';
-import { ICommissionInfo, GroupValidator } from 'controllers/chain/substrate/staking';
```

Comment:
This file is not merged properly. This is overwriting changes from staking-ui.

Reply:
Reverted back the changes, kindly check all files.